src/ontology/uberon-edit.obo

```diff
@@ -223738,6 +223738,17 @@ relationship: dc-contributor https://orcid.org/0000-0002-0819-0473 ! Paula Duek
 property_value: dcterms-date "2023-11-02T07:48:12Z" xsd:dateTime
 property_value: editor_note "The surface layer is formed by a dense collagen matrix with perpendicular aligned collagen fibers. The inner layer consists of a loosely disorganized collagen matrix with a high concentration of mesenchymal stem cells. Fully mature roots lack apical papilla." xsd:string {xref="PMID:18498881", xref="PMID:34026757"}
 
+[Term]
+id: UBERON:8480032
+name: neck of pancreas
+def: "The portion of the pancreas located at the junction of the head and body of the pancreas." [NCIT:C158551]
+synonym: "neck of the pancreas" EXACT [NCIT:C158551]
+synonym: "pancreatic neck" EXACT [FMA:14517]
+is_a: UBERON:0000064 ! organ part
+relationship: dc-contributor https://orcid.org/0000-0002-0819-0473 ! Paula Duek Roggli
+relationship: part_of UBERON:0001264 ! pancreas
```

Comment:
Consider adding the following axioms that are described in the definition:
- 'continuous with' some 'head of pancreas'
- 'continuous with' some 'body of pancreas'